pyproject.toml

```diff
@@ -1,3 +1,64 @@
+[project]
+name = "aggregator-api"
+version = "0.0.0"
+description = "devs.DC"
+readme = "README.md"
+requires-python = "==3.12.*"
+dependencies = [
+    "django==4.2.23",
```

Comment:
The approach I have taken here is that the root/top-level project is essentially `frontend`.

This is the approach from the previous pipenv implementation and also what we did in https://github.com/DemocracyClub/ec-api-proxy

I think you could also make a reasonable argument for saying that frontend should actually be a workspace, and the top-level project should just be a container for all the workspaces. If we do that, I think it requires a bit of restructuring so lets make it a follow up if we want to do that. I'm not 100% sure if doing that buys us very much though. It could just end up being churn.